kover-offline-runtime/src/main/java/kotlinx/kover/offline/runtime/api/KoverRuntime.java

```diff
@@ -42,6 +67,47 @@ public static List<ClassCoverage> collect(List<byte[]> classFiles) {
         return convertClasses(CoverageRuntime.collectClassfileData(classFiles));
     }
 
+    /**
+     * Save coverage binary report in file with ic format.
+     * If the file does not exist, it will be created. If a file with that name already exists, it will be overwritten.
+     * <p/>
+     * Calling this method is allowed only after all tests are completed. If the method is called in parallel with the execution of the measured code, the coverage value is unpredictable.
+     *
+     *
+     * @param fileName name of the file to save binary report
+     * @throws IOException in case of any error working with files
+     */
+    public static void saveReport(String fileName) throws IOException {
+        final File file = new File(fileName);
+
+        if (!file.exists()) {
+            file.getParentFile().mkdirs();
+            file.createNewFile();
+        }
+
+        try (DataOutputStream outputStream = new DataOutputStream(new BufferedOutputStream(new FileOutputStream(file)))) {
```

Comment:
IIRC, this idiom is not recommended to use because if the constructor of `BufferedOutputStream` throws an exception, the underlying `FileOutputStream` will not be closed (because variable `outputStream` is not constructed yet). Although we probably can ignore this problem, because constructors do not throw `IOException` and other exceptions are considered non-recoverable